benchmarks/Flocking/flocking.py

```diff
@@ -136,15 +137,15 @@ def make_agents(self):
             x = self.random.random() * self.space.x_max
             y = self.random.random() * self.space.y_max
             pos = np.array((x, y))
-            velocity = np.random.random(2) * 2 - 1
+            direction = np.random.random(2) * 2 - 1
```

Comment:
I just saw the docstring says "headings" and this needs to be replaced with "directions" as well.